libpoolprotocols/stratum/EthStratumClient.cpp

```diff
@@ -40,7 +40,7 @@ static void diffToTarget(uint32_t *target, double diff)
 }
 
 
-EthStratumClient::EthStratumClient(boost::asio::io_service & io_service, int worktimeout, int responsetimeout, string const & email, bool const & submitHashrate) : PoolClient(),
+EthStratumClient::EthStratumClient(boost::asio::io_service & io_service, int worktimeout, int responsetimeout, string email, bool submitHashrate) : PoolClient(),
```

Comment:
If you are passing `std::string` by value you should also move it later on in
```
m_email(std::move(email))
```

Reply:
Actually m_email should be no longer needed as it's a legacy of old -O argument.
Will drop it in next jobs.